### ISSUE TITLE
Cannot disable "skip import" on a GLTF file once it's enabled

### ISSUE THREAD

**Issue by hakro** [godotengine/godot#86613]
### Tested versions

4.2.1 Stable

### System information

Godot v4.2.1.stable - Ubuntu 22.04 - X11 - GLES3 (Compatibility) - Mesa Intel(R) Graphics (RPL-P) () - 13th Gen Intel(R) Core(TM) i7-13700H (20 Threads)

### Issue description

When importing a GLTF model, once you enable "Skip Import" from the "Advanced Import Settings", it becomes impossible to disable it, until its related `.glb.import` file is deleted from the filesystem

### Steps to reproduce

- Drag & Drop a glb model the 3D editor. It's there
- Double click a glb file to open the "Advanced Import Setting" screen
- Enable "Skip import" on the mesh, then reimport
- It disappears, as it should
- Go back to "Advanced Import Setting", and disable "Skip Import", then reimport.
- The mesh does not show up.
- Go back to "Advanced Import Setting", the "Skip Import" is still enabled, as if it was not saved from last time

A work around, is to navigate to your filesystem, and delete the `.glb.import` file. But obviously this will cause all your previous import settings to be lost for that model.

https://github.com/godotengine/godot/assets/3624853/cdf241b2-4af3-4f2c-9633-585caf47a9de

### Minimal reproduction project (MRP)

No need.
Any basic glb file with an empty project would do.

**Comment by jsjtxietian** [comment]
Yes, but if add disable "Skip Import", and then enable something like generate physics, then reimport, the mesh will be shown again.

**Comment by amarraff** [comment]
Just had an issue similar to this. Not sure if it's worth creating a separate issue since my situation seems to also be related to the importer not saving changes; it is also fixed by deleting its .glb.import file.

I changed the layers of some mesh instances within my .glb humanoid mesh file from layer 1 to layer 4. I also accidentally left a couple of the meshes with no layer.

I decided I wanted to change them back, so I set them all to layer 1, and reimported. Now, all but one of my meshes (the "head" mesh) in the file is on layer 1. If I set its layer to 4, layer 1 *and* layer 4, or any other combination, it works fine. But, it will not let me set the mesh to just layer 1.

Furthermore, this effect can be transferred. If I set the "head" mesh to layer 1 only, and set one of the "foot" meshes to layer 1 and 4, the head mesh will finally be only at layer 1 on reimport. But, now, the "foot" mesh cannot just be layer 1, and must be layer 4, 1 & 4, etc.

**Comment by DearFox** [comment]
Really annoying bug

**Comment by DearFox** [comment]
A quick solution for anyone who has encountered this:
open `*.glb.import` as a text file and search for `skip_import` so you can disable this.
![изображение](https://github.com/godotengine/godot/assets/59219907/753238dd-27dd-416c-a3c1-f75e3a0beef5)

**Comment by DearFox** [comment]
I'm not sure yet that this is the case, but I assume that the error must be somewhere in these files...
https://github.com/godotengine/godot/blob/7529c0bec597d70bc61975a82063bb5112ac8879/editor/import/3d/scene_import_settings.cpp
https://github.com/godotengine/godot/blob/7529c0bec597d70bc61975a82063bb5112ac8879/editor/import/3d/resource_importer_scene.cpp
Maybe I'm wrong, but I'll keep looking...
